thesis-gui/src/components/my-quizzes/QuizCreator.tsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import {
+    IonButton,
+    IonCol,
+    IonContent, IonGrid,
+    IonIcon,
+    IonInput,
+    IonItem,
+    IonList,
+    IonPage, IonRadio, IonRadioGroup, IonRow,
+    IonTextarea,
+    IonTitle
+} from "@ionic/react";
+import {PageHeader} from "../common/PageHeader";
+import {arrowBackOutline, arrowForwardOutline} from "ionicons/icons";
+
+const PAGE_NAME = "Quiz Creator";
+
+const AnswerInputItem: React.FC<{ answerNumber: number }> = ({ answerNumber }) => {
+    return (
+        <IonItem>
+            <IonInput placeholder={ `Enter answer ${answerNumber}` }></IonInput>
+            <IonRadio slot="end"></IonRadio>
+        </IonItem>
+    );
+}
+
+const AnswersInput: React.FC = () => {
+    const answerInputItems = Array.from({ length: 4 }, (_, i) => i + 1).map(answerNumber => {
```

Comment:
I didn't know that we can generate an Array of numbers in JS this way, so I learned something new :) . But to be honest, for such a small array `[0,1,2,3].map(...` looks more readable :) .